x-pack/legacy/plugins/epm/server/lib/elasticsearch/ingest_pipeline/ingest_pipelines.ts

```diff
@@ -119,7 +120,7 @@ async function installPipeline({
   // which we could otherwise use.
   // See src/core/server/elasticsearch/api_types.ts for available endpoints.
   await callCluster('transport.request', callClusterParams);
-  return { id: pipeline.name, type: 'ingest-pipeline' };
+  return { id: pipeline.name, type: AssetType.IngestPipeline };
```

Comment:
+1 on this change and the one with index template.

Suggestion for the future: Could you put it in a small separate PR and get it in quickly as its not directly related to this change? This makes both PR's smaller to review and quicker to be merged.